runtime/src/append_vec.rs

```diff
@@ -680,18 +681,6 @@ pub mod tests {
         );
     }
 
-    #[test]
-    fn test_set_file_empty_data() {
```

Comment:
This test now covered by https://github.com/solana-labs/solana/pull/7693/files#diff-657d1e7648a9fb74cb1582a9c48beb4bR566